dfastmi/gui/dialog_view_model.py

```diff
@@ -145,16 +148,16 @@ def qthreshold(self, value: float):
         self.qthreshold_changed.emit(self._qthreshold)
 
     @property
-    def reference_files(self) -> Dict[float, str]:
+    def reference_files(self) -> Dict[float, Path]:
         """
-        Dict[float, str]: The reference files.
+        Dict[float, Path]: The reference files.
         """
         return self._reference_files
 
     @property
-    def measure_files(self) -> Dict[float, str]:
+    def measure_files(self) -> Dict[float, Path]:
         """
-        Dict[float, str]: The measurement files.
+        Dict[float, Path]: The measurement files.
```

Comment:
same comment as above.

Reply:
Discussed with Ralph, we will set it back to str.